model_helper.py

```diff
@@ -335,6 +339,16 @@ def las_model_fn(features,
             output_dir=os.path.join(config.model_dir, run_name),
             summary_op=attention_summary)
         hooks = [eval_summary_hook]
+        if binf_embedding is not None:
+            with tf.name_scope('binf_image'):
+                binf_image = tf.tile(binf_embedding[None, :, :, None], [tf.shape(encoder_inputs)[0], 1, 1, 1])
```

Comment:
Why tile it for a whole batch? Isn't `tf.expand_dims(tf.expand_dims(binf_embedding, axis=0), axis=-1)` simpler?

Reply:
According to the [documentation](https://www.tensorflow.org/api_docs/python/tf/summary/image), "The images are built from tensor which must be 4-D with shape `[batch_size, height, width, channels]`"

Reply:
I tried double expand dims several times and it worked well.